src/api/python/genenums.py.in

```diff
@@ -51,13 +51,15 @@ r'''# distutils: language = c++
 from {basename} cimport {enum} as c_{enum}
 from enum import Enum
 
-class {enum}(Enum):
-    """The {enum} enum"""
-    def __new__(cls, value, doc):
+class DocEnum(Enum):
```

Comment:
Having `__new__` be inside the `Kind` enum yields a `'staticmethod' object is not callable`. I am not sure exactly why, but creating a parent class `DocEnum` seems to have solved the problem.